src/app/views/sidebar/history/History.tsx

```diff
@@ -39,7 +41,8 @@ export class History extends Component<IHistoryProps, any> {
 
 
   public componentDidMount = () => {
-    this.generateGroupedList(this.props.history);
+      this.generateGroupedList(this.props.history);
+      telemetry.trackEvent(TAB_CLICK_EVENT, { ComponentName: "History Tab" });
```

Comment:
technically this is when the component is coming up not when the tab is clicked.  any reason this wasn't added in the onclick event?

Reply:
I found out I could use the onLinkClick callback from IPivotProps interface.